fili-wikipedia-example/README.md

```diff
@@ -20,8 +20,16 @@ This example is an entirely self contained example that provides a Fili applicat
     mvn install
     mvn -pl fili-wikipedia-example exec:java
     ```
+    - Note that if your setup is different you can adjust it by changing the default parameters below
+        ```bash
+        # cd fili
+        # mvn install
+        mvn -pl fili-generic-example exec:java -Dbard__fili_port=9998 \
+        -Dbard__druid_coord=http://localhost:8081/druid/coordinator/v1 \
+        -Dbard__druid_broker=http://localhost:8082/druid/v2 \
+        ```
```

Comment:
Can we untab this section (backspace x4) to make this look like the [fili-generic-example](https://github.com/yahoo/fili/blob/master/fili-generic-example/README.md)?

Also `-Dbard__druid_broker=http://localhost:8082/druid/v2 \` -> `-Dbard__druid_broker=http://localhost:8082/druid/v2` (extra `\` here and in `fili-generic-example`)

Reply:
Odd thing I did not see this. Good catch!